scripts/config/lib/ngx_mruby/listing_redirect_path.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# ghetto require, since mruby doesn't have require
+eval(File.read('/app/bin/config/lib/nginx_config_util.rb'))
+
+USER_CONFIG = '/app/static.json'
+
+req         = Nginx::Request.new
+uri         = req.var.uri
+
+city, state, id = uri.match(%r{/listings/([A-z0-9\-]*)\/([A-z0-9\-]*)\/([0-9]*)$}mi).captures
+
+"storage-units-near-me/#{state.downcase}/#{city.downcase}/#{id.downcase}"
```

Comment:
I think we'll need to include the other listing details, not just ID so that there aren't duplicate pages in the cache